src/dex-ui-components/Pool/Withdraw.tsx

```diff
@@ -0,0 +1,189 @@
+import { SettingsIcon } from "@chakra-ui/icons";
+import {
+  Box,
+  HStack,
+  Flex,
+  Heading,
+  Spacer,
+  IconButton,
+  Button,
+  Text,
+  TableContainer,
+  Table,
+  Tbody,
+  Tr,
+  Td,
+} from "@chakra-ui/react";
+import { ChangeEvent, useState, useCallback } from "react";
+import { WalletConnectionStatus } from "../models/wallet.model";
+import { TokenInput } from "../TokenInput";
+
+export interface WithdrawProps {
+  walletConnectionStatus: WalletConnectionStatus;
+  poolLpDetails: LPTokenDetails;
+  poolLiquidityDetails: PoolLiquidityDetails;
+  onWithdrawClick: (lpAmount: number) => void;
+  onInputAmountChange?: (lpAmount: number) => void;
+  disableWithdrawButton?: boolean;
+}
+
+interface LPTokenDetails {
+  tokenSymbol: string;
+  userLpAmount: number;
+  userLpPercentage: string;
+}
+
+interface TokenLiquidityDetails {
+  tokenSymbol: string;
+  poolLiquidity: number;
+  userProvidedLiquidity: number;
+}
+
+interface PoolLiquidityDetails {
+  firstToken: TokenLiquidityDetails;
+  secondToken: TokenLiquidityDetails;
+}
+
+interface PoolMetricDisplayProps {
+  label: string;
+  value: string;
+}
+
+const PoolMetricDisplay = (props: PoolMetricDisplayProps) => {
+  const { label, value } = props;
+
+  return (
+    <Flex flexDirection={"column"}>
+      <Text fontSize={"10px"} lineHeight={"12px"}>
+        {label}
+      </Text>
+      <Text fontSize={"14px"} lineHeight={"17px"}>
+        {value}
+      </Text>
+    </Flex>
+  );
+};
+
+const WithdrawComponent = (props: WithdrawProps) => {
+  const {
+    walletConnectionStatus,
+    poolLpDetails,
+    poolLiquidityDetails,
+    onWithdrawClick,
+    onInputAmountChange,
+    disableWithdrawButton,
+  } = props;
+
+  const [localWithdrawState, setLocalWithdrawState] = useState({
+    lpInputAmount: "0",
+  });
+
+  const handleInputAmountChange = (event: ChangeEvent<HTMLInputElement>) => {
+    if (onInputAmountChange) onInputAmountChange(+event.target.value);
+    setLocalWithdrawState({ ...localWithdrawState, lpInputAmount: event.target.value });
+  };
+
+  const getPortionOfBalance = useCallback(
+    (portion: "half" | "max") => {
+      setLocalWithdrawState({
+        ...localWithdrawState,
+        lpInputAmount:
+          portion === "half" ? (+poolLpDetails.userLpAmount / 2).toString() : poolLpDetails.userLpAmount.toString(),
+      });
+    },
+    [localWithdrawState, poolLpDetails.userLpAmount]
+  );
+
+  return (
+    <HStack>
+      <Box data-testid="pool-component" bg="white" borderRadius="24px" width="100%" padding="1rem">
```

Comment:
`data-testid` should be set to a unique identifier for this component (maybe, `withdraw-liquidity-component`). `pool-component`pool-component is already being used.